### ISSUE TITLE
Multiple currencies

### ISSUE THREAD

**Issue by hgpit** [pluginsGLPI/order#222]
Are there any plans to support multiple currencies?

We purchase IT assets from all over the world, so we naturally get invoiced/billed int various currencies. Would be great to be able to store this information.

Thanks.

**Comment by retrack** [comment]
yes, that would be very useful, even if the exchange rate management is manual at first

**Comment by jeffallen** [comment]
If we accept that the data model of GLPI is mono-currency, then "only" the following would need to be done to address this:

- set the database currency, update all places where costs are displayed to include the currency name
- update all places where *new* cost data is entered into the system to include a "from-currency" popup, which should default to the database currency, and a date field called "cost effective date" or something, which is the date that should be used for currency rate lookup.
- have a setting showing where to get rates from (i.e. https://exchangeratesapi.io/#pricing_plan)
- before inserting new data into the database, convert using the effective date, the from currency, and the amount.
